gt-profile.yml

```diff
@@ -0,0 +1,106 @@
+BagIt-Profile-Info:
+  BagIt-Profile-Identifier: https://ocr-d.github.io/gt-profile.json
+  BagIt-Profile-Version: '1.2.0'
+  Source-Organization: OCR-D
+  External-Description: BagIt profile for OCR line Ground Truth
+  Contact-Name: Konstantin Baierer
+  Contact-Email: konstantin.baierer@sbb.spk-berlin.de
+  Version: 0.1
+Bag-Info:
+  Bagging-Date:
+    required: false
+  Source-Organization:
+    required: false
+  Gt-Transcription-Extension:
+    required: false
+    default: '.gt.txt'
+  Gt-Transcription-Media-Type:
+    required: false
+    default: 'text/plain'
+  Gt-Transcription-Directory:
+    required: false
+    default: 'text'
+  Gt-Transcription-Normalization:
+    required: false
+    default: 'NFKC'
```

Comment:
Not sure if we should have a default here. I'd opt for `required` without default. Too much depends on a correct setting.

Reply:
In addition, the specification should require "utf-8" encoding for the GT text somewhere to make those normalizations work, right?

Reply:
If requiring a value I'd add a custom one as there might be some sets that aren't normalized for whatever reasons.